extensions/VisualEditor/wikia/modules/ve/ui/widgets/ve.ui.WikiaFocusWidget.js

```diff
@@ -33,7 +33,8 @@ ve.ui.WikiaFocusWidget = function VeUiWikiaFocusWidget( surface ) {
 	this.$body = this.$( this.getElementDocument() ).find( 'body:first' );
 	this.$window = this.$( this.getElementWindow() );
 	this.$surface = surface.$element;
-	this.$pageHeaderElements = this.$( '#WikiaPageHeader' ).children( ':not( h1 )' );
+	this.$pageHeaderElements = this.$( '#WikiaPageHeader' ).children( ':not( .header-container  )' ).add( '.header-tally' );
```

Comment:
That class is added for sure, code in this pr depends on it.
I know we didn't use it originally because content in page header wasn't completely predictable, I'm not sure if it changed. (I don't remember details)